airflow-provider-fivetran/operators/fivetran.py

```diff
@@ -3,11 +3,12 @@
 import time
 
 import requests
-import pendulum
 
 from airflow.models import BaseOperator
 from airflow.utils.decorators import apply_defaults
 
+from airflow-provider-fivetran.airflow-provider-fivetran.hooks.fivetran import FivetranHook
```

Comment:
I believe `.hooks.fivetran` should work. `airflow-provider-fivetran.airflow-provider-fivetran` is not going to work with the installed package.